website/templates/project/settings.mako

```diff
@@ -132,15 +130,14 @@
                         <form id="selectAddonsForm">
 
                             % for category in addon_categories:
-
                                 <%
-                                    addons = [
-                                        addon
-                                        for addon in addons_available
-                                        if category in addon.categories
-                                    ]
+                                    addons = []
```

Comment:
If we are treating wiki as a core OSF feature, then this change should be happening on the backend, i.e. at the view layer around here: https://github.com/CenterForOpenScience/osf.io/blob/91078b2875b38f5f3f6df0698540e5084fd06be2/website/project/views/node.py#L281 , rather than doing all this logic here.